CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to Zarb project
+
+A warm welcome for considering contributing to Zarb project!
+
+## Getting Started
+
+Contributions are made to this repo via Issues and Pull Requests (PRs).
+
+<TO_BE_DEFINED>
```

Comment:
might be a good idea to link discord channel here

Reply:
@loreanvictor What is your idea?

Reply:
templates look good. CONTRIBUTING.md of course needs work, I would recommend outlining project structure, build and testing process (so people can also run tests on their own dev environments), and code conventions that people should follow (basically stuff that you would need to explain on a per-PR basis, if this file did not exist).